packages/selected-network-controller/tests/SelectedNetworkController.test.ts

```diff
@@ -263,18 +264,10 @@ describe('SelectedNetworkController', () => {
     afterEach(() => {
       jest.clearAllMocks();
     });
-    it('should throw an error when passed "metamask" as domain arg', () => {
-      const { controller } = setup();
-      expect(() => {
-        controller.setNetworkClientIdForDomain('metamask', 'mainnet');
-      }).toThrow(
-        'NetworkClientId for domain "metamask" cannot be set on the SelectedNetworkController',
-      );
-      expect(controller.state.domains.metamask).toBeUndefined();
-    });
+
     describe('when the useRequestQueue is false', () => {
       describe('when the requesting domain is not metamask', () => {
```

Comment:
nit we can probably get rid of this nested describe block `when the requesting domain is not metamask` since the test applies whether or not the request domain is `MetaMask`